BGAnimations/ScreenEvaluation common/Panes/Pane4/default.lua

```diff
@@ -32,13 +32,13 @@ end
 -- if players have disabled W5 or W4+W5, there will be a smaller range
 -- of judgments that could have possibly been earned
 local num_judgments_available = 5
-local worst_window = PREFSMAN:GetPreference("TimingWindowSecondsW5")
+local worst_window = GetTimingWindow(5)
```

Comment:
GetTimingWindow (num_judgments_available) seems a but more apt

I think we should add a function that returns "NumJudgmentsAvailable" as we reference the number 5 elsewhere and it would be nice for it to be descriptive.

Reply:
The function would likely just return 5, but it's nice not having magic number :)

Reply:
done!